docs/source-2.0/additional-specs/smoke-tests.rst

```diff
@@ -0,0 +1,194 @@
+.. _smoke-tests:
+
+===========
+Smoke Tests
+===========
+
+Smoke tests are small, simple tests intended to uncover large issues by ensuring
+basic functionality works as expected. In Smithy, these tests are used to make
+sure clients can successfully make a request to a live service, and that the
+service responds as expected.
```

Comment:
nit: there are some phrasing inconsistencies in this PR for "service responds as expected", "service should respond to a given request", "get the right kind of response"